packages/core/src/lib/light_push/index.ts

```diff
@@ -56,7 +56,12 @@ class LightPush extends BaseProtocol implements ILightPush {
     pubSubTopic: string
   ): Promise<PreparePushMessageResult> {
     try {
-      if (!isSizeValid(message.payload)) {
+      if (!message.payload || message.payload.length === 0) {
```

Comment:
I guess if we continue extending validation for `payload` then maybe makes sense to have `validatePayload` function?

Reply:
Yes I considered it and went for this small PR to fix the issue and then started https://github.com/waku-org/js-waku/pull/1643 to fix wider issue around validation.